redirects.yaml

```diff
@@ -855,6 +855,13 @@ usn/rss\.xml: "/security/notices/rss.xml"
 telecommunications/?: "/telco"
 telcos/?: "/telco"
 
+# Community
+community/canonical/?: "/community/governance/canonical"
+community/code-of-conduct/?: "/community/governance/code-of-conduct"
+community/debian/?: "/community/governance/debian"
+community/diversity/?: "/community/governance/diversity"
+community/mission/?: "/community/governance/mission"
+
```

Comment:
This could also be done in the [index topic](https://discourse.ubuntu.com/t/community/33115), so the community docs team can manage it. But in terms of functionality it's the same.